wolox-books/src/app/screens/shopping-cart/components/list/list.component.html

```diff
@@ -0,0 +1,17 @@
+<modal [(open)]="open"> 
+  <h1 class="modal-title">Tu carrito de compras</h1>
+  <div *ngFor="let book of books" class="modal-container">
+    <p class="modal-text">{{book.title}} - {{book.author}}</p>
+    <button type="button" (click)="deleteFromShoppingCart(book.id)">
+      <img src="../../../assets/delete.png" alt="delete" class="modal-delete">
```

Comment:
close tag

Reply:
ERROR in Errors parsing template: Void elements do not have end tags "img" ("tems">
    <img src="../../../../../assets/no-items.png" alt="no-items" class="empty-shopping-cart">[ERROR ->]</img>
    <div class="empty-text">
      <p class="empty-title">EL CARRITO EST&Aacute; VAC&Iacute;O<"): /Users/jorgediaz/Documents/projects/trainings/jd-angular/wolox-books/src/app/screens/shopping-cart/components/list/list.component.html@9:93